### PR TITLE
Add BigBed file generation, Remove BEDGraph step, add skip params

### PR DESCRIPTION
**Pull request by lwratten**
Hello, a few changes for the downstream file generation:
- Merged `BAMToBed12` and `Bed12ToBigWig` into one process `BAMToBigWig`
- No saving of `BEDGraph` file to output dir
- Added `BAMToBigBed` Process
- Added params for skipping these processes (each case has been tested)
   `--skip_visualisation` (maybe `--skip_vis` is better)
    `--skip_bigbed` 
    `--skipbigwig`
- Updated docs etc. to reflect these changes

Many thanks for contributing to nf-core/nanoseq!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [x] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/nanoseq branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/nanoseq)
 - [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [ ] Make sure your code lints (`nf-core lint .`).
 - [x] Documentation in `docs` is updated
 - [ ] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/nanoseq/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -651,10 +691,53 @@ if (params.skip_alignment) {
...
+     * STEP 13 - Convert BED12 to BigBED
+     */
+    process Bed12ToBigBed {
+        tag "$sample"
+        label 'process_medium'
+        publishDir path: "${params.outdir}/${params.aligner}/bigbed/", mode: 'copy',
+            saveAs: { filename ->
+                          if (filename.endsWith(".bb")) filename
+                    }
+        input:
```
Comment (drpatelh) [inline main.nf]:
```suggestion
        when:
        !params.skip_bigbed
  
        input:
```

File: main.nf [hunk 2]
```diff
@@ -651,10 +691,53 @@ if (params.skip_alignment) {
...
+        set file(fasta), file(sizes), val(sample), file(bed12) from ch_bed12
+
+        output:
+        set file(fasta), file(sizes), val(sample), file("*.bb") into ch_bigbed
+
+        script:
+        """
+        bedToBigBed $bed12 $sizes ${sample}.bb
```
Comment (drpatelh) [inline main.nf]:
```suggestion
        bedToBigBed $bed12 $sizes ${sample}.bigBed
```

File: main.nf [hunk 3]
```diff
@@ -651,10 +691,53 @@ if (params.skip_alignment) {
...
+        label 'process_medium'
+        publishDir path: "${params.outdir}/${params.aligner}/bigbed/", mode: 'copy',
+            saveAs: { filename ->
+                          if (filename.endsWith(".bb")) filename
+                    }
+        input:
+        set file(fasta), file(sizes), val(sample), file(bed12) from ch_bed12
+
+        output:
+        set file(fasta), file(sizes), val(sample), file("*.bb") into ch_bigbed
```
Comment (drpatelh) [inline main.nf]:
```suggestion
        set file(fasta), file(sizes), val(sample), file("*.bigBed") into ch_bigbed
```

File: main.nf [hunk 4]
```diff
@@ -651,10 +691,53 @@ if (params.skip_alignment) {
...
+
+    /*
+     * STEP 13 - Convert BED12 to BigBED
+     */
+    process Bed12ToBigBed {
+        tag "$sample"
+        label 'process_medium'
+        publishDir path: "${params.outdir}/${params.aligner}/bigbed/", mode: 'copy',
+            saveAs: { filename ->
+                          if (filename.endsWith(".bb")) filename
```
Comment (drpatelh) [inline main.nf]:
```suggestion
                          if (filename.endsWith(".bigBed")) filename
```

File: main.nf [hunk 5]
```diff
@@ -727,7 +811,7 @@ ${summary.collect { k,v -> "            <dt>$k</dt><dd><samp>${v ?: '<span style
 }
 
 /*
- * STEP 12 - MultiQC
+ * STEP 15 - MultiQC
  */
 process MultiQC {
     publishDir "${params.outdir}/multiqc", mode: 'copy'
```
Comment (drpatelh) [inline main.nf]: Add FastQC output to MultiQC too.

File: nextflow.config [hunk 6]
```diff
@@ -31,10 +31,16 @@ params {
   save_align_intermeds = false
   skip_alignment = false
 
+  // Options: Visualisation
+  skip_visualisation = false
```
Comment (drpatelh) [inline nextflow.config]:
```suggestion
```

File: docs/usage.md [hunk 7]
```diff
@@ -30,10 +30,15 @@
   * [`--aligner`](#--aligner)
   * [`--save_align_intermeds`](#--save_align_intermeds)
   * [`--skip_alignment`](#--skip_alignment)
+* [Visualisation file generation](#visualisation-file-generation)
+  * [`--skip_visualisation`](#--skip_visualisation)
```
[the remark this answers is not in the document]
Comment (lwratten) [reply docs/usage.md]: Agree :)

File: main.nf (hunk 5 above)
Comment (lwratten) [reply main.nf]: Done!

**Comment by lwratten** [comment]
Okay hopefully that's a bit better

**Comment by drpatelh** [comment]
Much better. Thank you 👍

File: main.nf [hunk 8]
```diff
@@ -651,10 +691,56 @@ if (params.skip_alignment) {
...
+    /*
+     * STEP 12 - Convert BAM to BED12
+     */
+    process BAMToBed12 {
+        tag "$sample"
+        label 'process_medium'
+
+        when:
+        !params.skip_bigbed
+        input:
```
Comment (drpatelh) [inline main.nf]: Line space here.

Comment (drpatelh) [reply main.nf]: Minor bug that can be updated later 👍 Going to merge.

**Pull request merged**